GsaGH/Helpers/GH/ReplaceParam.cs

```diff
@@ -36,5 +38,63 @@ public static string UnsupportedValue(GH_ObjectWrapper ghTypeWrapper) {
       type = type.Replace("Goo", string.Empty);
       return type;
     }
+
+    public static void UpdateReleaseBool6Parameter(this GH_ComponentParamServer parameters) {
```

Comment:
why not 2 methods? one for input and one for output

Reply:
@DominikaLos @psarras I prefer least call. Single function call here will have least chance of error like updating input but   missing to update output parameter. However, I am open for your suggestion.Kinldy confirm again and I will update code.